tensorflow_probability/python/mcmc/transformed_kernel_test.py

```diff
@@ -71,7 +96,7 @@ def setUp(self):
     super(TransformedTransitionKernelTest, self).setUp()
     self.dtype = np.float32
 
-  def test_support_works_correctly_with_HMC(self):
+  def test_support_works_correctly_with_hmc(self):
```

Comment:
are these changes because pylint complained?

Reply:
yeah, all the similar caps lock changes are because of that

Reply:
final thoughts: keep lowercase